test/parallel/test-crypto-keygen.js

```diff
@@ -603,6 +603,25 @@ const sec1EncExp = (cipher) => getRegExpForPEM('EC PRIVATE KEY', cipher);
       passphrase: 'top secret'
     });
   }));
+
+  // Test async elliptic curve key generation with 'jwk' encoding
+  generateKeyPair('ec', {
+    namedCurve: 'P-384',
+    publicKeyEncoding: {
+      type: 'spki',
+      format: 'jwk'
+    },
+    privateKeyEncoding: {
+      type: 'pkcs8',
+      format: 'jwk'
+    }
+  }, common.mustSucceed((publicKey, privateKey) => {
+    assert.strictEqual(typeof publicKey, 'object');
+    assert.strictEqual(typeof privateKey, 'object');
+    assert.strictEqual(publicKey.x, privateKey.x);
+    assert.strictEqual(publicKey.y, privateKey.y);
+    assert.strictEqual(publicKey.kty, privateKey.kty);
```

Comment:
```suggestion
    assert(!publicKey.d);
    assert(privateKey.d);
    assert.strictEqual(publicKey.kty, 'EC');
    assert.strictEqual(publicKey.kty, privateKey.kty);
    assert.strictEqual(publicKey.crv, 'P-384');
    assert.strictEqual(publicKey.crv, privateKey.crv);
```

I'm suspecting once this is added the tests will start failing, that's because this JS part (https://github.com/nodejs/node/blob/v16.4.2/lib/internal/crypto/keys.js#L193-L213) is not involved.

That makes sure that

- only JWK applicable key types are handled
- only JWK registered EC curves are handled
- JWK EC and OKP keys have their `crv` property set properly